### PR TITLE
add figma code connect to Web UI

### PR DESCRIPTION
**Pull request by robphoenix**

### CODE REVIEW

File: packages/web-ui/src/Link/Link.figma.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import figma from '@figma/code-connect';
+import { Link } from './Link';
+
+figma.connect(
```
Comment (jordmccord) [inline packages/web-ui/src/Link/Link.figma.tsx:5]: Worth adding to .npmignore?

Comment (robphoenix) [reply packages/web-ui/src/Link/Link.figma.tsx:5]: ah, yes, definitely, good catch 🙌🏻

**Comment by robphoenix** [comment]
@jordmccord do you know anything about this in the broken CI check?

image

**Comment by jordmccord** [comment]
@robphoenix I do, but I don't know why it's doing that 🤔 will have to investigate. Feel free to merge either way

**Review by jordmccord — APPROVED** [review]

**Pull request merged**